homeassistant/components/xbee/binary_sensor.py

```diff
@@ -3,12 +3,14 @@
 
 from homeassistant.components.binary_sensor import BinarySensorEntity
 
-from . import DOMAIN, PLATFORM_SCHEMA, XBeeDigitalIn, XBeeDigitalInConfig
-
-CONF_ON_STATE = "on_state"
-
-DEFAULT_ON_STATE = "high"
-STATES = ["high", "low"]
+from . import (
```

Comment:
Why not make a constants file if you want to move them?

Reply:
I could if that's the convention we want. @MartinHjelmare

Reply:
Constants that are shared among multiple package modules are preferred to be put in a const.py module in the package. Local constants can stay in the module where they are needed. :+1: